### PR TITLE
Cu xfbj0p have a editorconfig file for all environment rabbil yasar

### PR DESCRIPTION
**Pull request by rabbilyasar**

### CODE REVIEW

**Comment by rabbilyasar** [comment]
Task linked: [CU-xfbj0p have a editorconfig file for all environment](https://app.clickup.com/t/xfbj0p)

File: backend/requirements.txt [hunk 1]
```diff
@@ -10,6 +10,5 @@ pycodestyle==2.7.0
 pyflakes==2.3.1
 pytz==2021.1
 sqlparse==0.4.1
-toml==0.10.2
```
Comment (ibrahimyaacob92) [inline backend/requirements.txt]: what is this toml for ? I know what it is but where are you planning to use them ?

**Review by ibrahimyaacob92 — DISMISSED** [review]

File: backend/requirements.txt (hunk 1 above)
Comment (rabbilyasar) [reply backend/requirements.txt]: This has been removed

Comment (ibrahimyaacob92) [reply backend/requirements.txt]: damn.. my eyes arent sharp this time haha

Comment (Muntaaha) [reply backend/requirements.txt]: Ahh Nice work!

**Review by Muntaaha — DISMISSED** [review]

File: backend/requirements.txt (hunk 1 above)
Comment (rabbilyasar) [reply backend/requirements.txt]: nvm I think we will be needing it for autopep8

**Pull request merged**